desktop/static/setup.tsx

```diff
@@ -83,6 +85,10 @@ export default async function setup(argv: any) {
     updaterEnabled: argv.updater,
     launcherEnabled: argv.launcher,
     launcherMsg: argv.launcherMsg,
+    suppressPluginUpdateNotifications:
```

Comment:
Is there documentation that needs to be updated as well? I'm not sure how end users are supposed to know about config settings for flipper.

Reply:
there are no docs for the config.json file AFAIK...would be nice if there were but that's probably a separate PR...